docs/notebook-demos/reading-in-sentinel-data.ipynb

```diff
@@ -0,0 +1,244 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Reading in Sentinel-2 Images\n",
+    "\n",
+    "Sentinel-2 is an observation mission developed by the European Space Agency to monitor the surface of the Earth [official website](http://www.esa.int/Our_Activities/Observing_the_Earth/Copernicus/Sentinel-2).\n",
+    "Sets of images are taken of the surface where each image corresponds to a specific wavelength. These images can provide useful data for a wide variety of industries, however, the format they are stored can prove difficult to work ith. This being, `JPEG 2000` (file extension `.jp2`), an image compression\n",
+    "format for JPEGs that allows for improved quality and compression ratio.\n",
+    "\n",
+    "## Why Use GeoPySpark\n",
+    "\n",
+    "There are few libraries and/or applications that can work with `jp2`s and big data, which can make processing large amounts of sentinel data difficult. However, by using GeoPySpark in conjunction with the tools available in Python, we are able to read in and work with large sets of sentinel imagery."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting the Data\n",
+    "\n",
+    "Before we can start this tutorial, we will need to get the sentinel images. All sentinel data can be found on Amazon's S3 service, and we will be downloading it straight from there.\n",
+    "\n",
+    "We will downloading three different `jp2`s that represent the same area and time in different wavelengths. These being: Aerosol detection (443 nm), Water vapor (945 nm), and Cirrus (1375 nm). Why these three bands? It's because of the resolution of the image, which determines what bands are represented best. For this example, we will be\n",
```

Comment:
We will **download** three different `jp2`s that represent the same area and time in different wavelengths: Aerosol detection (443 nm), Water vapor (945 nm), and Cirrus (1375 nm).  These bands are chosen because they are all in the same 60m resolution.  The tiles we will be working with cover the eastern coast of Corsica taken on January 4th, 2017.